sklearn/ensemble/_forest.py

```diff
@@ -1321,15 +1329,19 @@ class RandomForestRegressor(ForestRegressor):
            The default value of ``n_estimators`` changed from 10 to 100
            in 0.22.
 
-    criterion : {"squared_error", "mse", "mae"}, default="squared_error"
+    criterion : {"squared_error", "mse", "mae", "poisson"}, default="squared_error"
```

Comment:
```suggestion
    criterion : {"squared_error", "mse", "mae", "poisson"}, \
            default="squared_error"
```
The line was too long, hence again pep8/linter error.

Reply:
You could install `pre-commit`, see https://scikit-learn.org/stable/developers/contributing.html#how-to-contribute. This way, git won't allow you commits that introduce linter/pep8 errors and warn you where this happens.

Reply:
Apologies for the back-and-forth @lorentzenchr , I've troubleshooted all the CI errors. I've actually had precommit installed and running with each of my commits this whole time - I think its test suite isn't as comprehensive as the CI.